parts/kubernetesmastercustomscript.sh

```diff
@@ -298,6 +304,7 @@ ensureDocker
 configNetworkPolicy
 ensureKubelet
 extractKubectl
+ensureJournal
```

Comment:
I found, if i don't do reload and restart journal, it will not take my customized config.